app/controllers/spree/api/checkouts_controller_decorator.rb

```diff
@@ -0,0 +1,23 @@
+module Sprangular::ApiCheckoutControllerDecorator
+  def quick_update
+    load_order(true)
+    authorize! :update, @order, order_token
+    
+    Sprangular::OrderUpdater.update(@order, request, params)
+
+    respond_with(@order, default_template: 'spree/api/orders/show')
+  end
+
+  def refresh_shipping_rates
+    load_order(true)
+    authorize! :update, @order, order_token    
+
+    @order.create_proposed_shipments
+    @order.updater.update_shipments    
+    @order.update_from_params(params, permitted_checkout_attributes, request.headers.env)      
```

Comment:
Line is too long. [95/80]<br>Unnecessary spacing detected.<br>Trailing whitespace detected.